container/README.md

```diff
@@ -0,0 +1,63 @@
+# Docker Image Builder
+
+There are several docker image files in the sub directories of current directory. Each sub directory contains Dockerfile of an image. Please find description of each image as follows.
+
+|  Sub directory | Image name  | Description  | 
+|---|---|---|
+|  ccnp-device-plugin | ccnp-device-plugin  | Device plugin for Intel® TDX |
```

Comment:
Suggest to change "Device plugin for Intel® TDX"  to "CCNP device plugin", since it is not the official device plugin for tdx. And it is mainly used to passthrough device node and mount required directory in ccnp Daemonsets.

Reply:
Updated. Thanks.